pkgs/development/python-modules/mwcli/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, docopt
+, fetchPypi
+, para
+, pythonRelaxDepsHook
+}:
+
+buildPythonPackage rec {
+  pname = "mwcli";
+  version = "0.0.3";
+
+  src = fetchPypi {
```

Comment:
why not `fetchFromGitHub`?

Reply:
Because unfortunately there are no releases/tags on the repo.